Global/JetBrains.gitignore

```diff
@@ -44,6 +44,16 @@ cmake-build-*/
 # IntelliJ
 out/
 
+# Eclipse
```

Comment:
These rules are already captured in [`Global/Eclipse.gitignore`](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore).